src/activity_item_handling.cpp

```diff
@@ -2928,6 +2928,12 @@ int get_auto_consume_moves( player &p, const bool food )
                 // its frozen
```

Comment:
```suggestion
                // it's frozen
```
(Yes, I'm compulsive.)